### ISSUE TITLE
TMUX status bar Window color are automatically inverted until the mouse is clicked AND each time the Window is selected via C-a w (bug?)

### ISSUE THREAD

**Issue by carboncrystal** [tmux/tmux#3499]
### Describe the problem
TMUX status bar Window color are automatically inverted until the mouse is clicked on that Windows (clicked in the TMUX status bar) **AND** each time the Window is selected `via C-a w` the previous Window have the color automatically inverted again (compared to the colors declared in .tmux.conf).

**1. As you can see in the .tmux.conf shown below I have stated so:** 
$ Color status bar
set -g status-fg white
set -g status-bg blue

**2. I also declared inactive Window as**
$ Inactive window color:
set -g window-status-style fg=white,bg=blue

**3. I also declared Window active as**
$ Active window color:
set-window-option -g window-status-current-style fg=black,bg=yellow

### So based on my configuration in .tmux.conf, the expected result in my TMUX bar should be this:
Status bar: bg=bule,fg=white
Active window: bg=yellow,fg=black
Inactive Window (same as status bar): fg=white,bg=blue
![expected-result](https://user-images.githubusercontent.com/106466414/225842269-ae72fcf2-9935-48e9-989c-d51d8f20d9fe.png)

**But** when I start TMUX for the first time and then via Tmuxinator I load my session via my YML file the COLOURS ARE REVERSED, i.e. my inactive windows have bg=white and fg=blue (the exact opposite declared in `.tmux.conf`):
![1 when-tmux-start](https://user-images.githubusercontent.com/106466414/225844066-2c2d8937-78c9-415b-91af-c4931aafa91a.png)
This behavior returns to normal once I click in the TMUX status bar on the respective windows. 
See here an example where I clicked in the TMUX status bar on "MySecondWindow" and then clicked again on "MyFirstWindow". You can see that the "MySecondWindow" color has returned to the colors declared in .tmux.conf:
![2 mouse-click-my-second-window](https://user-images.githubusercontent.com/106466414/225847710-cce89229-f10d-4551-96eb-bea52a426a2e.png)

But the bug comes back into the picture again when I select windows via `C-a w`: the previous window reverses its colours again.
It's not just related to these colors, I tried other colors for inactive windows (for example bg=black and fg=yellow) and the same bug is reproduced.

I use Starship as terminal emulator, but I don't think it has anything to do with it because I disabled Starship from .bashrc (after sourced) and the bug still reproduces.

Please tell me any solution how to get rid of this bug as it distracts me from my daily activity.

### .tmux.conf
```
# TMUX Custmization
unbind C-b
set-option -g prefix C-a
# Alt +1 - Activate Command to all Panes & border color changed to red
bind-key -n M-1 setw synchronize-panes on \;  set-window-option status-bg red \; display-message "Pane Sync ON"
# Alt +2 - Exist from Command to all Panes, border color back to deafaut
bind-key -n M-2 setw synchronize-panes off \;  set-window-option status-bg default \; display-message "Pane Sync OFF"
# Allow us to reload our Tmux configuration while using Tmux
bind-key r source-file ~/.tmux.conf \; display "Reloaded Tmux Conf!"
# Allow the arrow key to be used immediately after changing windows
set-option -g repeat-time 0
# Set window notifications
set -g monitor-activity on
# Update files on focus (using for neovim)
# set -g focus-events on
# Status update interval
set -g status-interval 1
# Increase scroll-back history
set -g history-limit 5000
# Highlight window when it has new activity
setw -g monitor-activity on
set -g visual-activity on
# Re-number windows when one is closed
set -g renumber-windows on
set-option -g allow-rename off
# Terminal TMUX screen color
set -g default-terminal "screen-256color"
set -g terminal-overrides "screen-256color"
# set -g pane-border-format "#{pane_index} #{pane_title}"
set -g pane-border-status top
set -g mouse on

###########################
# ALIAS TMUX
###########################

set -s command-alias[6] mouseon='set -g mouse on'  # Activate mouse
set -s command-alias[7] mouseoff='set -g mouse off' # Deactivate mouse
set -s command-alias[8] viewalias='show-options command-alias' # View all aliases

###########################
# STATUS BAR TMUX
###########################

# Color status bar
set -g status-fg white
set -g status-bg blue
# No separator between windows
# set-window-option -g window-status-separator ''


# Show session and window in left status bar
set-option -g default-command bash
set-option -g status-left " ⚡️ #S "
# Inactive window color
set -g window-status-style fg=white,bg=blue
# Active window color
set-window-option -g window-status-current-style fg=black,bg=yellow
#set -g status-style bg=blue
set -g status-left-length 85
set -g status-right-length 85
set -g status-right " %l:%M • %d %b %Y %p"

# Set refresh interval for status bar
# set -g status-interval 30
###########################
# PLUGINS TMUX
###########################

# List of TMUX Plugins
# set -g @plugin 'tmux-plugins/tpm'
# set -g @plugin 'tmux-plugins/tmux-sensible'
# set -g @plugin 'tmux-plugins/tmux-resurrect'
# set -g @plugin 'tmux-plugins/tmux-continuum'
# Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
# run '~/.tmux/plugins/tpm/tpm'
```

### Required information

* tmux version: tmux 3.2a
* Platform: Linux x86_64
* OS: Ubuntu 22.04.2 LTS
* $TERM > Outside tmux: xterm-256color | Inside tmux: screen-256color
* Logs from tmux (updated again):
[tmux-server-257247.zip](https://github.com/tmux/tmux/files/11007390/tmux-server-257247.zip) - compressed .zip because it is 36 MB, over the GitHub upload limit of 25 MB
[tmux-client-257244.log](https://github.com/tmux/tmux/files/11007397/tmux-client-257244.log)

**Comment by nicm** [comment]
I cannot see any of your window names (MyFirstWindow, MySecondWindow etc) in this log, are you sure the log shows the same thing as your screenshots?

**Comment by carboncrystal** [comment]
Hi!

I have now recreated the log and edited the above post and I upload the new log files. 100% I used the windows above.

**Comment by nicm** [comment]
Are you sure you used the right log files?
~~~~
$ grep My tmux-server-237760.log
$
~~~~

**Comment by carboncrystal** [comment]
Hi @nicm !
I've figured out what the problem was with the log file. According to your templeate to create an Issue I have complied in precisely with: * "Logs from tmux (`tmux kill-server; tmux -vv new`)". But in my case I should have loaded my session via the Tmuxinator YML file and attached to it, and not just "tmux -vv new".
I have now recreated the log again and edited the first post and uploaded the new log files (zipped due to GitHub upload limit).

Now "grep My tmux-server-257247.log" shows the windows log.
